assets/style.css

```diff
@@ -199,177 +224,194 @@ header h1 {
+  font-weight: bold;
+}
+
+.close:hover,
+.close:focus {
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+/* results tiles */
+
 .columns {
   display: flex;
+  flex-wrap: flex;
   flex-direction: row;
   justify-content: center;
 }
 
-#wordSubmit {
-    color: black;
-    font-size: 1.25rem;
-    font-weight: 500;
-    background-color: var(--primary-green);
-    background-color: rgb(247, 236, 198);
-    margin: 6px 4px 12px 0;
-    padding: 4px;
-    border-radius: 6px;
-    text-align: center;
-    width: 10rem; 
-}
-
-
-
-/* #colorSubmit {
-    text-align: center;
-    font-size: 1.2rem;
-    color: black;
-    background-color: rgb(189, 208, 236);
-    margin: 6px 0 12px 0;
-    padding: 4px;
-    border-radius: 6px;
-    min-width: 7rem;
-} */
+.column1 {
+  margin-left: 0.75rem;
+}
 
-#startOver {
-    text-align: center;
-    font-size: 1.25rem;
-    font-weight: 500;
-    color: black;
-    background-color: rgb(203, 190, 233);
-    margin: 6px 0 12px 0;
-    padding: 4px;
-    border-radius: 6px;
-    width: 10rem;
+.column5 {
+  margin-right: 0.75rem;
 }
 
-#wordText {
-    background-image: none;
-    font-size: 1.2rem;
-    margin-top: 0.2rem;
-    margin-bottom: 0.6rem;
+#searchedWord,
+#word1,
+#word2,
+#word3,
+#word4 {
+  font-size: 1.5rem;
+  color: black;
 }
 
-/* The Modal (background) */
-.modal {
-    display: none; /* Hidden by default */
-    position: fixed; /* Stay in place */
-    z-index: 1; /* Sit on top */
-    padding-top: 100px; /* Location of the box */
-    left: 0;
-    top: 0;
-    width: 100%; /* Full width */
-    height: 100%; /* Full height */
-    overflow: auto; /* Enable scroll if needed */
-    background-color: rgb(0,0,0); /* Fallback color */
-    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
-  }
-  
-  /* Modal Content */
-  .modal-content  {
-    
-    background-color: #fefefe;
-    margin: auto;
-    padding: 20px;
-    border: 1px solid #888;
-    width: 80%;
-  }
-  .modal-content p{
-    color: #000;
-    font-weight: bold;
-  }
-  
-  /* The Close Button */
-  .close {
-    color: #aaaaaa;
-    float: right;
-    font-size: 28px;
-    font-weight: bold;
-  }
-  
-  .close:hover,
-  .close:focus {
-    color: #000;
-    text-decoration: none;
-    cursor: pointer;
-  }
-  
-  #retryCard {
-    max-width: 30%;
-    min-height: fit-content;
-    max-height: 12rem;
-  }
-
-  .column1 {
-    margin-left: 0.75rem;
-  }
-
-  .column5 {
-    margin-right: 0.75rem;
-  }
-
-  #searchedWord, #word1, #word2, #word3, #word4 {
-    font-size: 1.5rem;
-    color: black;
-  }
-
-  .colorTile {
-    text-shadow: rgb(172, 171, 171) 2px 2px 2px;
-    color: black;
-    font-size: 1.25rem;
-    font-weight: 500;
-  }
-
-  .colorTile p {
-    font-size: 1.5rem;
-  }
-
-.column1, .column2, .column3, .column4, .column5 {
+.colorTile {
+  text-shadow: rgb(172, 171, 171) 2px 2px 2px;
+  color: black;
+  font-size: 1.25rem;
+}
+
+.colorTile p {
+  font-size: 1.5rem;
+}
+
+.column1,
```

Comment:
instead of targeting these each individually you can probably give them a shared class to make this more dry

Reply:
that's on HTML